src/util/mapbox.js

```diff
@@ -103,11 +103,11 @@ export class RequestManager {
     }
 
     normalizeSpriteURL(url: string, format: string, extension: string, accessToken?: string): string {
-        const urlObject = parseUrl(url);
-        if (!isMapboxURL(url)) {
-            urlObject.path += `${format}${extension}`;
-            return formatUrl(urlObject);
+        if (!isMapboxURL(url))
+        {
+            return url + `${format}${extension}`;
```

Comment:
CI is failing because the we have ESLint configured to prefer templates to concatenation.

```suggestion
            return `${url}${format}${extension}`;
```